pages/docs/users-and-permissions/seat-types.mdx

```diff
@@ -0,0 +1,36 @@
+# Seat Types and User Roles
+
+Hashboard uses a seat-based licensing model with two distinct seat types: Editor and Collaborator.
+Each user in your Hashboard instance occupies either an Editor or Collaborator seat based on their assigned roles and permissions.
+
+> **Note:** The Editor and Collaborator seat types share names with our default system-defined roles
+> (see [Managing Users and Roles](/docs/users-and-permissions/managing-users-and-roles)),
+> but seat type and role-name are distinct concepts. Hashboard's default role named "Editor" that comes with every new project can be modified to no longer be considered an
+> Editor seat type, and similarly, a role named "Collaborator" can be adjusted to have permissions that qualify it as an Editor seat type.
+
+## Understanding seat types
+
+### Editor Seats
+
+Editors are users who can modify data models and have broader control over the analytics environment.
+A user is considered an Editor if they have any of the following permissions:
+
+- `CREATE_DATA_MODEL`
```

Comment:
Should build permissions be on here as well?

Reply:
Good question. Ultimately this is a business decision. @trucklos @vndrewlee 

So the proposal would to also include `CREATE_DATA_MODEL_REPO_BUILD` to the list of permissions that qualifies a role/user as an Editor seat

Reply:
Correct me if I'm wrong, but it seems to me that even if you have `CREATE_DATA_MODEL_REPO_BUILD`, you still need to have the appropriate create / update model permissions to actually change them? https://github.com/gleannyc/glean/blob/2040369d21f0bf4047aed40c68ee53db68ae157b/glean/services/data_ops/model_conversions.py#L413

Reply:
@dse0 you are correct, and @luchodev just verified this in the context of the status update permission. So it makes sense that CREATE_DATA_MODEL_REPO_BUILD does not make a role an editor seat-role